packages/@okta/vuepress-site/docs/reference/csi-delauth-hook/index.md

```diff
@@ -0,0 +1,536 @@
+
+The hook supports a few operations. It first allows you to import a user into Okta after a successful authentication. From there it allows delegated authentication of the user to the external source, and fetches the latest profile of the user from the external source.
+
+Each of these operations corresponds to an individual inline hook invocation, and each corresponding request is tagged with a `requestType` property in the request headers.
+
+The following sections describe each of these operations and the objects in their requests and responses.
+
+## User attributes
+
+To support user lifecycle and credential lifecycle operations, we expect the external source to provide certain attributes:
+
+- `status`: The status of the user, with a value of ACTIVE or DISABLED.
+- `passwordExpiryTime`: The time when the user's password expires, in the form of Epoch time.
+- `lastUpdate`: When the user's profile was last updated in the external source, in the form of Epoch time.
+
+## Delegated authentication request
+
+This type of request has the following `requestType`:
+
+- `user.authenticate`
+
+### Objects in the request from Okta
+
+The outbound call from Okta to your external service includes the following objects in its JSON payload:
+
+#### data.context.credential
+
+Provides two name-value pairs, one for identifying the subject user to authenticate, and the other for providing the password of that user.
+
+#### data.action
+
+The current default action that Okta takes if your external service sends an empty HTTP 204 response. You can override the default action by returning a `commands` object in your response specifying the action to take.
+
+### Objects in the response you send
+
+The `commands` and `error` objects that you can return in the JSON payload of your response are defined as follows:
+
+#### commands
+
+The `commands` object is where you can provide commands to Okta. It is an array, allowing you to send multiple commands. Each array element needs to consist of the following name-value pair:
+
+| Property | Description                                           | Data Type       |
+|----------|-------------------------------------------------------|-----------------|
+| type     | One of the [supported commands](#supported-commands) | String          |
+| value    | The parameter to pass to the command                  | [value](#value) |
+
+##### Supported commands
+
+The following command is supported for the delegated authentication request:
+
+| Command                         | Description                                                                                                              |
+|---------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| `com.okta.action.update`          | Specify the result of the authentication, either success or one of the causes of failure. |
+
+##### value
```

Comment:
in the fetch profile and auth + fetch request types, you added #####com.okta.action.update before ##### value. Can you follow the same pattern here

Reply:
Yeah there's a discrepancy that needs to be removed. After looking at the other inline hook (registration for example), they don't explicitly list the command name before value, so I've updated the other two sections to do the same.